.circleci/config.yml

```diff
@@ -69,6 +69,4 @@ workflows:
   securedrop_ci:
     jobs:
       - docs-lint
-      - staging-test:
-          requires:
-            - docs-lint
+      - staging-test
```

Comment:
Why the removal of `docs-lint`? Do we want `staging-test` to continue if doc linting fails?

Reply:
I was trying to shave 2 minutes to be honest ... i mean, you are right... we dont necessarily want staging-test to continue if doc linting fails.... dang it @redshiftzero and your sound logic

Reply:
Agreed with @redshiftzero, let's keep it. We're considering moving all the linting logic to non-AWS-ified Circle CI anyway (#2217).

Reply:
Okay I re-based and removed the commit - to restore previous test dependency flow. 👍